conversations/v3/common.yaml

```diff
@@ -56,7 +56,7 @@ info:
 
     **Retries**
 
-    The tyntec application will retry to post the request 2 more times in the case your
+    The tyntec system will retry to post the request 2 more times in the case your
```

Comment:
This is to be discussed. I've replaced every occurrence of `tyntec application` with `tyntec system`. My understanding is that the term `application` is a relic of v2 and in v3 it might be confusing.